docs/components/dropdown.md

```diff
@@ -0,0 +1,570 @@
+    { label: 'Top', to: { path: '/components/dropdown' } }
+  ]"
+/>
+```
+
+### appearance
+
+Use this prop to customize the default trigger KButton's [appearance prop](/components/button.html#appearance).
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" appearance="danger" :items="deepClone(defaultItems)" />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  appearance="danger"
+  :items="items"
+/>
+```
+
+### showCaret
+
+Use this prop if you would like the trigger button to display a caret icon.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :items="deepClone(defaultItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  show-caret
+  :items="items"
+/>
+```
+
+### icon
+
+:::warning NOTE
+This prop will be removed in the `9.0.0-beta.0` release. Icons can be provided in the `default` slot
+:::
+
+A string for the `KIcon` to be displayed on the dropdown button with or in place of the button label.
+
+<ClientOnly>
+  <KDropdown icon="cogwheel" :items="deepClone(defaultItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown icon="cogwheel" :items="items" show-caret />
+```
+
+### width
+
+The width of the dropdown body (defaults to `auto`). Currently we support numbers (will be converted to `px`), `auto`, and percentages for width.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :items="deepClone(defaultItems)" width="700" />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  width="700"
+  :items="items"
+/>
+```
+
+### kpopAttributes
+
+Use the `kpopAttributes` prop to configure the KPop [props](/components/popover) dropdown.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :kpop-attributes="{ placement: 'leftStart' }" :items="deepClone(defaultItems)" />
+</ClientOnly>
+
+```html
+<KDropdown 
+  trigger-text="Documentation"
+  :kpop-attributes="{ placement: 'leftStart' }"
+  :items="items"
+/>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with [`disabledTooltip` prop](#disabledtooltip).
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<ClientOnly>
+  <KDropdown
+    trigger-text="Documentation"
+    disabled
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItems)"
+  />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  disabled
+  disabled-tooltip="You can't click me"
+  :items="items"
+/>
+```
+
+### isSelectionMenu
+
+Use this prop when a visual indication of the currently selected menu item is needed. 
+
+By default the dropdown has no notion of "selection":
+
+<ClientOnly>
+  <KDropdown trigger-text="Select region" :items="deepClone(selectionMenuItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Select region"
+  :items="[{ label: 'US (United States)', value: 'us' }, { label: 'FR (France)', value: 'fr' }]"
+  show-caret
+/>
+```
```

Comment:
Ok so I literally just clicked through this example again and didn't notice it was the wrong one 😓 

Can we remove the example of "not adding the prop" from this prop section? It's extremely confusing

Reply:
I guess you're right, it's better to remove this example here (I was trying to follow the way we had this section before).